cli/flox-rust-sdk/src/models/manifest/typed.rs

```diff
@@ -863,6 +863,7 @@ pub enum ManifestError {
 #[derive(Debug, Clone, Serialize, Deserialize, Default, PartialEq, Eq)]
 #[cfg_attr(test, derive(proptest_derive::Arbitrary))]
 pub struct Include {
+    #[serde(default)]
```

Comment:
nonblocking question: for everything other than `[include]`, is the conclusion from your other PRs that we want to use an `Option` for fields that need to be merged? What's the reasoning for not using an `Option` here?

Reply:
> for everything other than `[include]`, is the conclusion from your other PRs that we want to use an `Option` for fields that need to be merged?

Yeah

> What's the reasoning for not using an `Option` here?

It felt different for two reasons:

1. It doesn't get rendered into the lockfile so there's no issues with compatibility (across versions) or presentation (e.g. `flox list --config`)
2. It currently only contains a single vec which is the primary indicator of whether composition is in use; but that rationale (and the skip serializing logic) could change if we add other fields in the future